lib/vanagon/platform.rb

```diff
@@ -111,6 +111,7 @@ def initialize(name) # rubocop:disable Metrics/AbcSize
       @sort ||= "sort"
       @copy ||= "cp"
       @cross_compiled ||= false
+      @vendor_build_tools ||= false
```

Comment:
Regardless of what we call this, shouldn't this default to `true`? The idea is that "most platforms probably use native vendor-provided upstream toolchains", right?